### PR TITLE
Added documentation for `getImportedRouteNames()`

### PR DESCRIPTION
**Pull request by hmvien**
Make sure you've checked off all these things before submitting:

- [x] This pull request isn't for a company's fork, it's intended for the upstream Slate shared by everybody.
- [ ] This pull request is submitted to the `dev` branch.
- [ ] If it makes frontend changes, this pull request has been tested in the latest version of Firefox, Chrome, IE, and Safari.

### CODE REVIEW

File: source/includes/getting-started.md [hunk 1]
```diff
@@ -126,6 +163,7 @@ public class SetupFoxtrotSDKActivity extends AppCompatActivity {
     // Marshmallow requires certain permissions to be granted at run-time
     final List<String> missingPermissions = getMissingPermissions(REQUIRED_PERMISSIONS);
     if (!missingPermissions.isEmpty()) {
+      // When a user select an action on the prompt, the callback onRequestPermissionsResult() will be invoked.
```
Comment (loganakamatsu) [inline source/includes/getting-started.md]: select -> selects

Comment (isaaczafuta) [inline source/includes/getting-started.md]: select -> selects

File: source/includes/getting-started.md [hunk 2]
```diff
@@ -371,6 +413,33 @@ Now that we’ve created a Route, it can be imported like this:
 FoxtrotSDK.getInstance().addRoute(route);
 ```
 
+### Ensuring a route is not imported more than once
+
+The `FoxtrotSDK` <b>will not</b> deduplicate or merge duplicate routes for you. If you imported the same route twice, then two copies of the routes will be
```
Comment (isaaczafuta) [inline source/includes/getting-started.md]: Remove the 'ed' on imported so the tense agrees?

File: source/includes/getting-started.md [hunk 3]
```diff
@@ -371,6 +413,33 @@ Now that we’ve created a Route, it can be imported like this:
...
+saved and optimized by Foxtrot. This scenario is not ideal since it creates confusion for the driver. In order to resolve this problem, it is recommended
```
Comment (isaaczafuta) [inline source/includes/getting-started.md]: 'it is recommended to' -> 'you should'

File: source/includes/getting-started.md [hunk 4]
```diff
@@ -415,10 +415,10 @@ FoxtrotSDK.getInstance().addRoute(route);
 
 ### Ensuring a route is not imported more than once
 
-The `FoxtrotSDK` <b>will not</b> deduplicate or merge duplicate routes for you. If you imported the same route twice, then two copies of the routes will be
-saved and optimized by Foxtrot. This scenario is not ideal since it creates confusion for the driver. In order to resolve this problem, it is recommended
-to name a route with a unique identifier. Once this requirement is met, you may use the `FoxtrotSDK.getImportedRouteNames()` method to obtain
-a `Map<String, String>` object mapping from `route id` to `route name` for all imported routes seen by the `FoxtrotSDK`. Here is a sample code demonstrating
+The `FoxtrotSDK` <b>will not</b> deduplicate or merge duplicate routes for you. If you import the same route twice, then two copies of the routes will be
+saved and optimized by Foxtrot. This scenario is not ideal since it creates confusion for the driver. In order to resolve this problem, you should
+name yours routes with unique identifiers. Once this requirement has been met, you may use the `FoxtrotSDK.getImportedRouteNames()` method to obtain
```
Comment (isaaczafuta) [inline source/includes/getting-started.md]: yours -> your

**Pull request merged**